jssource/jsmin.php

```diff
@@ -45,4 +45,50 @@ protected function jsParser() {
         require_once('jssource/Minifier.php');
         return Minifier::minify($this->text);
 	}
+
+    /**
+     * Join and minify JS files
+     *
+     * @param array $jsFiles an 'array' of js files
+     *
+     * @author Jose C. Massón <jose@gcoop.coop>
+     *
+     * @return Minified JS file path
+     */
+    static public function joinAndMinifyJSFiles($jsFiles)
```

Comment:
try to avoid the static calls.

Reply:
Done in commit e3c2726e86236ba107e2a466406bfb7d081eebfc